web/src/portico/signup.ts

```diff
@@ -49,6 +65,7 @@ $(() => {
     );
 
     $("#registration, #password_reset, #create_realm").validate({
+        onkeyup: false,
```

Comment:
Are there any downsides to disabling this event?

Reply:
@amanagr , I don't see any such specific downsides while testing. 
If you find any bugs, please let me know. 
Will try to address it. 
Thanks!